README.md

```diff
@@ -23,7 +23,12 @@ spec:
   interval: 12h
   url: oci://ghcr.io/gimlet-io/capacitor-manifests
   ref:
-    semver: ">=0.1.0"
+    tag: v-cosign-test5
```

Comment:
Currently specifying the exact tag since other tags/version are not signed, once we agree on what next we either, sign all version after `=0.1.0` and put back `semver: ">=0.1.0"` or we keep using tags ..